tests/end-to-end/test_api_user_login.py

```diff
@@ -33,15 +33,18 @@ def test_login_with_password(self):
 
         print(df_ver)
 
-        opts = {"server_host": "datafed-server-test.ornl.gov"}
+        datafed_domain = os.environ.get("DATAFED_DOMAIN")
+        opts = {"server_host": datafed_domain}
+        print("Creating API")
         df_api = API(opts)
-
+        print("API created")
         username = "datafed99"
         password = os.environ.get("DATAFED_USER99_PASSWORD")
 
         count = 0
         while True:
             try:
+                print(f"username: {username}, password: {password}")
```

Comment:
I assume that this is just because it is a test, but it is printing a password.

Reply:
Goot catch, this should be removed though.